dc/council/code/titles/1/sections/1-609.54.xml

```diff
@@ -9,12 +9,16 @@
   <para>
     <num>(b)</num>
     <text>An individual appointed to the Management Supervisory Service pursuant to this section may be paid severance pay upon separation for non-disciplinary reasons according to the length of the individual’s employment with the District government as follows:</text>
+    <text>
+      <table>
+        <tr><th>Length of Employment</th><th>Maximum Severance</th></tr>
+        <tr><td>Up to 6 months</td><td>2 weeks of the employee’s basic pay</td></tr>
+        <tr><td>6 months to 1 year</td><td>4 weeks of the employee’s basic pay</td></tr>
+        <tr><td>1 to 3 years</td><td>8 weeks of the employee’s basic pay</td></tr>
+        <tr><td>More than 3 years</td><td>0 weeks of the employee’s basic pay.</td></tr>
```

Comment:
I assume the change from **10 weeks** to **0 weeks** was unintentional, and should probably be fixed

Reply:
Good catch. Fixed. Thank you.